include/crow/middleware.h

```diff
@@ -0,0 +1,312 @@
+#pragma once
+
+#include "crow/http_request.h"
+#include "crow/http_response.h"
+#include "crow/utility.h"
+
+#include <tuple>
+#include <type_traits>
+#include <iostream>
+#include <utility>
+
+namespace crow
+{
+
+    /// Local middleware should extend ILocalMiddleware
+    struct ILocalMiddleware
```

Comment:
For Interface. I looked for naming conventions in the source, found `crow::ILogHandler`. Maybe it could be called just LocalMiddleware, which however doesn't hint that it has no functionality

Reply:
Makes sense, thanks for explaining (`ILogHandler` was written before I started working on the project), and thanks for not changing conventions.